spec/requests/pages_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe PagesController, type: :controller do
+  describe 'index' do
+    before do
+      get :index
+   it "Gets last 4 articles" do
+    articles = Article.last(4)
```

Comment:
here you should test the  respone
you are sending get request and you are expecting response with valid status and template index that is render

maybe something like this 

```
describe 'GET index' do
  before do
    get :index
  end

  it “returns a successful response” do
    expect(response).to be_successful
  end

  it "renders the index template" do
    get :index
    expect(response).to render_template("index")
  end
end
```

more details here 
https://medium.com/just-tech/rspec-controller-or-request-specs-d93ef563ef11